src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/NEUStatsCommand.kt

```diff
@@ -179,6 +187,32 @@ class NEUStatsCommand {
         return builder
     }
 
+    private fun appendRepoStats(builder: DiscordMarkdownBuilder): DiscordMarkdownBuilder {
+        val apiData = NotEnoughUpdates.INSTANCE.config.apiData
+        val contains = builder.toString().lines().any { it == "# Repo Stats" }
```

Comment:
Why check for this here? Instead you could just replace the lines in in `#appendStats` with a call to this method.